python/pyproject.toml

```diff
@@ -16,7 +16,7 @@ whylogs-sketching = ">=3.4.1.dev3"
 protobuf = ">=3.19.4"
 importlib-metadata = { version = "<4.3", python = "<3.8" }
 typing-extensions = { version = ">=3.10", markers = "python_version < \"4\""}
-whylabs-client = "^0.6.2"
+whylabs-client = "^0.6.5"
```

Comment:
Thanks for making the fix, but you need to at least update the poetry.lock file's whylabs-client version since its at 0.6.4 it will fail the CI setup.